docs/pages/quickstart.rst

```diff
@@ -42,99 +69,179 @@ If you want to create or manage a configuration file manually, refer to the
+        print(patient)
+        for device in patient.devices:
+            print(' ', device)
 
-JSON Endpoints
-**************
+        print('')
 
-An accessor can be used to iterate over paginated data from the respective JSON endpoint:
+
+:class:`~runeq.resources.patient.get_all_patients` returns a :class:`~runeq.resources.patient.PatientSet`,
+which can be serialized as a list of dictionaries, e.g. to save the metadata to a file:
 
 .. code-block:: python
 
-    for result in accel.iter_json_data():
-         print(result.keys())
+    import json
+
+    with open('patients.json', 'w') as f:
+        json.dump(patients.to_list(), f, indent=4)
 
 
-To override the default query parameters, use keyword arguments:
+You can also convert a :class:`~runeq.resources.patient.PatientSet` to a collection of
+devices (a :class:`~runeq.resources.patient.DeviceSet`). This may be more convenient for
+a columnar data format, like a `pandas <https://pandas.pydata.org/>`_ DataFrame.
 
 .. code-block:: python
 
-    for text in accel.iter_json_data(device_id='patient-ABC,device-456'):
-        pass  # do something
+    import pandas as pd
 
+    devices = patients.devices
+    devices_df = pd.DataFrame(devices.to_list())
 
-CSV Endpoints
-*************
 
-An accessor can also iterate over paginated data from the respective CSV endpoint.
+Fetch Timeseries Data
+*********************
 
-Here, we use the accessor to build up a `pandas <https://pandas.pydata.org/>`_ DataFrame,
-containing the complete result set.
+Use :class:`~runeq.resources.stream_metadata.get_patient_stream_metadata` to get
+a :class:`~runeq.resources.stream_metadata.StreamMetadataSet` with details about
+a particular patient's data. If you're interested in a more specific set of streams,
+the function accepts additional filters.
 
 .. code-block:: python
 
-    import io
-    import pandas as pd
+    from runeq.resources.stream_metadata import get_patient_stream_metadata
+
+    patient_id = "abc123"
+    patient_streams = get_patient_stream_metadata(patient_id)
+    print(f'Found {len(patient_streams)} streams')
+
+Once you have a :class:`~runeq.resources.stream_metadata.StreamMetadataSet`,
+you can use the **filter** operation to get a more specific subset of streams:
+
+.. code-block:: python
+
+    # Filter for data collected from a particular device
+    device_id = "eb#8c31"
+    device_streams = patient_streams.filter(device_id=device_id)
+
+    # Filter by broad category
+    neural_streams = patient_streams.filter(category="neural")
+
+    # Specify multiple arguments to find streams that match
+    # all criteria
+    neural_device_streams = patient_streams.filter(
+        category="neural",
+        device_id=device_id,
+    )
 
-    df = pd.DataFrame()
-    for text in accel.iter_csv_text():
-        page_df = pd.read_csv(io.StringIO(text))
-        df.append(page_df)
+    # Use a custom filter function
+    import time
 
+    def in_last_two_weeks(stream) -> bool:
```

Comment:
This is a nice set of examples, and I love the in_last_two_weeks function. It seems like a good fit for the quickstart.